controllers/articles.controller.js

```diff
@@ -9,3 +9,14 @@ exports.getArticleById = (req, res, next) => {
       next(err);
     });
 };
+
+// All articles
+exports.getAllArticles = (req, res, next) => {
+  getArticles()
+    .then((articles) => {
+      res.send({ articles });
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
```

Comment:
Good controller.  Slightly similar name to your model. Consider using more different names / different naming convention for controllers and models (e.g. getArticles for controllers, selectArticles for models).